tests/src/Unit/MetricsTest.php

```diff
@@ -94,54 +96,26 @@ public function testAbstractMetricBase()
         $this->assertEmpty($groups);
     }
 
-    /**
-     * Covers \EdisonLabs\Metrics\Metric\AbstractMetricPercentage
-     */
-    public function testAbstractMetricPercentage()
```

Comment:
Why remove this test @jkribeiro ?

Reply:
We removed the `AbstractMetricPercentage` class, so removing the test as well. 

We removed the "percentage" class because we don't need specific "metrics type", the metric class should be generic and returns any kind of value.